packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+| permissionType | Permission to add to the Role        | URL          | [Permission type](#permission-types)) | TRUE     |
+
+#### Response parameters
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+#### Request example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/roles/cr0Yq6IJxGIr0ouum0g3/permissions/okta.users.manage"
+```
+
+#### Response example
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+### Delete permission
+<ApiLifecycle access="beta" />
+
+<ApiOperation method="delete" url="/api/v1/iam/roles/${roleIdOrLabel}/permissions/${permissionType}" />
+
+Deletes a permission from an existing Role
+
+#### Request parameters
+
+| Parameter      | Description                          | Param Type   | DataType                              | Required |
+| :------------- | :----------------------------------- | :----------- | :------------------------------------ | :------- |
+| roleIdOrLabel  | `id` or `label` of the Role          | URL          | String                                | TRUE     |
+| permissionType | Permission to remove from the Role   | URL          | [Permission type](#permission-types)) | TRUE     |
+
+#### Response parameters
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+#### Request example
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/roles/cr0Yq6IJxGIr0ouum0g3/permissions/okta.users.manage"
+```
+
+#### Response example
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+### Delete role
+<ApiLifecycle access="beta" />
+
+<ApiOperation method="delete" url="/api/v1/iam/roles/${roleIdOrLabel}" />
+
+Deletes a Custom Role
+
+#### Request parameters
+
+| Parameter       | Description                          | Param Type   | DataType                              | Required |
+| :-------------- | :----------------------------------- | :----------- | :------------------------------------ | :------- |
+| roleIdOrLabel   | `id` or `label` of the Role          | URL          | String                                | TRUE     |
+
+#### Response parameters
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+#### Request example
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/roles/UserCreator"
+```
+
+#### Response example
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+## Resource set operations
+<ApiLifecycle access="beta" />
+These operations allow the creation and manipulation of Resource Sets as custom collections of resources. Resource Sets are used to assign [custom Roles](#custom-role-operations) to administrators, scoped to the designated resources..
```

Comment:
"...to administrators scoped to the designated resources." <- removed comma and extra period